src/views/Home.vue

```diff
@@ -333,10 +343,48 @@ export default class Home extends Vue {
           this.error = "There was an error in adding the document.";
           return;
         });
-      this.confirm = "";
+        let day = ""
+        switch (this.time.getDay()) {
+          case 0:
+            day = "Sunday";
+            break;
+          case 1:
+            day = "Monday"
+            break;
+          case 2:
+            day = "Tuesday"
+            break;
+          case 3:
+            day = "Wednesday"
+            break;
+          case 4:
+            day = "Thursday"
+            break;
+          case 5:
+            day = "Friday"
+            break;
+          case 6:
+            day = "Saturday"
+        }
+        const postData =
+        {
+          time: this.roundToQuarter(this.time),
+          cardio: this.cardio,
+          weights: this.weights
+        }
+        const options = {
+          headers: {'Content-Type': 'application/json'}
+        }
+        let url = process.env.VUE_APP_UPDATE_GYM_HISTORICAL_AVERAGES_API + '?id=' + current_gym + '&day=' + day;
+        Axios.post(url, postData, options).then(() => {
+          // SUCCESSFUL POST REQUEST :)
+        }).catch(err => {
+          throw err
+        })
+        this.confirm = "";
     } else {
-      this.error = "Please enter a value";
-      return;
+        this.error = "Please enter a value";
+        return;
```

Comment:
Might need to install [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or something in VS Code